syntax = "proto2";

package recursive_dependency_embedding;

import "sequence_node.proto";

message ScoredTree {
    required float score = 1;
    required SequenceNode tree = 2;
}